packages/dma-contracts/contracts/core/OperationsRegistry.sol

```diff
@@ -29,41 +29,45 @@ contract OperationsRegistry {
   }
 
   /**
-   * @notice Stores the Actions that constitute a given Operation
-   * @param newOwner The address of the new owner of the Operations Registry
-   */
-  function transferOwnership(address newOwner) public onlyOwner {
-    owner = newOwner;
-  }
-
-  /**
-   * @dev Emitted when a new operation is added or an existing operation is updated
+   * @dev Emitted when a new operation is added.
    * @param name The Operation name
+   * @param operationHash A hashed value of all action hashes ( concatenated )
    **/
-  event OperationAdded(bytes32 indexed name);
+  event OperationAdded(string name, bytes32 operationHash);
 
   /**
-   * @notice Adds an Operation's Actions keyed to a an operation name
-   * @param operation Struct with Operation name, actions and their optionality
+   * @notice Adds an operation under specific name for a specific ordered list of actions
+   * @dev Each action is stored in a ServiceRepository under a specific hash.
+   * To get the operationHash, one should concatenated all hashes of the actions
+   * that are used in the specific operation and hash them using keccak256.
+   * The order of the actions matters since different order will result in different hash value
+   * hence difference operation.
+   * @param name Name of the operation that is going to be stored
+   * @param operationHash A hash of all action hashes ( concatenated )
    */
-  function addOperation(StoredOperation calldata operation) external onlyOwner {
-    operations[operation.name] = operation;
+  function addOperation(string memory name, bytes32 operationHash) external onlyOwner {
```

Comment:
This is breaking the public interface and at least 2 files are affected:
- `operations-registry.ts` which I believe is used in tests and deployment system
- `tasks/operation-registry` which is one of the tools I did.

Fix should be trivial though